manticore/manticore.py

```diff
@@ -140,15 +157,16 @@ class Manticore(object):
     :type args: list[str]
     :ivar dict context: Global context for arbitrary data storage
     '''
-
-
-    def __init__(self, binary_path, args=None):
+    def __init__(self, binary_path, args=None, disasm='capstone'):
         assert os.path.isfile(binary_path)
 
         args = [] if args is None else args
 
+        self._check_disassembler_present(disasm)
+        self._disasm = disasm
         self._binary = binary_path
         self._binary_type = binary_type(binary_path)
+        # yan's comment
```

Comment:
Referring to  _# XXX(yan) '_args' will be removed soon; exists currently to ease porting_